xclim/indices/run_length.py

```diff
@@ -409,6 +453,61 @@ def windowed_run_count(
     return out
 
 
+def windowed_max_run_sum(
+    da: xr.DataArray,
+    window: int,
+    dim: str = "time",
+    freq: str | None = None,
+    ufunc_1dim: str | bool = "from_context",
+    index: str = "first",
+) -> xr.DataArray:
+    """Return the maximum sum of consecutive float values for runs at least as long as the given window length.
+
+    Parameters
+    ----------
+    da : xr.DataArray
+        Input N-dimensional DataArray (boolean).
+    window : int
+        Minimum run length.
+        When equal to 1, an optimized version of the algorithm is used.
+    dim : str
+      Dimension along which to calculate consecutive run (default: 'time').
+    freq : str
+      Resampling frequency.
+    ufunc_1dim : Union[str, bool]
+        Use the 1d 'ufunc' version of this function : default (auto) will attempt to select optimal
+        usage based on number of data points. Using 1D_ufunc=True is typically more efficient
+        for DataArray with a small number of grid points.
+        Ignored when `window=1`. It can be modified globally through the "run_length_ufunc" global option.
+    index : {'first', 'last'}
+        If 'first', the run length is indexed with the first element in the run.
+        If 'last', with the last element in the run.
+
+    Returns
+    -------
+    xr.DataArray, [int]
+        Total number of `True` values part of a consecutive runs of at least `window` long.
+    """
+    # TODO: implement ufunc function
+    # ufunc_1dim = use_ufunc(ufunc_1dim, da, dim=dim, index=index, freq=freq)
+
+    # if ufunc_1dim:
+    #     out = windowed_max_run_sum_ufunc(da, window, dim)
+
+    if window == 1 and freq is None:
+        out = da.max(dim=dim)
```

Comment:
Should this not be: 
```suggestion
    if window == 1 and freq is None:
        out = rse(da, dim=dim, index=index).max(dim=dim)
```
?